stable/polaris/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: Validation of best practices in your Kubernetes clusters
 name: polaris
-version: 4.2.3
-appVersion: "4.2"
+version: 5.0.0
+appVersion: "5.0"
```

Comment:
I don't think the version of polaris is changing, so we should keep this the same I think